### PR TITLE
Convert enum choices to integer choices in Daily Rounds

### LINKED ISSUE #2460: Convert enum choices to integer choices in Daily Rounds
https://github.com/ohcnetwork/care/blob/0a630f1fa247e0e77bae0908bb6dd64b8a2550e7/care/facility/models/daily_round.py#L32-L122

### PR DESCRIPTION
**Pull request by praffq**
## Proposed Changes

- [x] Update roundsType 
- [x] Update ConsciousnessType
- [x] Update PupilReactionType
- [x] Update LimbResponseType
- [x] Update  RythmnType
- [x] Update  VentilatorInterfaceType
- [x] Update  VentilatorModeType
- [x] Update VentilatorOxygenModalityType
- [x] Update  InsulinIntakeFrequencyType

### Associated Issue

- fixes #2460 

### Architecture changes

- Remove this section if not used

## Merge Checklist

- [x] Tests added/fixed
- [x] Linting Complete

_*Only PR's with test cases included and passing lint and test pipelines will be reviewed*_

@ohcnetwork/care-backend-maintainers @ohcnetwork/care-backend-admins

### CODE REVIEW

**Comment by sainak** [comment]
@DraKen0009 update the pr title

File: care/facility/api/viewsets/patient.py [hunk 1]
```diff
@@ -220,7 +219,9 @@ def filter_by_bed_type(self, queryset, name, value):
     )
     ventilator_interface = CareChoiceFilter(
         field_name="last_consultation__last_daily_round__ventilator_interface",
-        choice_dict=VENTILATOR_CHOICES,
+        choice_dict={
```
Comment (praffq) [inline care/facility/api/viewsets/patient.py:222]: Please check this once and let me know if there is some better way to do this.

File: care/facility/api/viewsets/daily_round.py [hunk 2]
```diff
@@ -26,7 +26,7 @@ def filter_rounds_type(self, queryset, name, value):
         values = value.split(",")
```
Comment (rithviknishad) [inline care/facility/api/viewsets/daily_round.py]: Just some minor nitpicks

```suggestion
        values = value.upper().split(",")
```

File: care/facility/api/viewsets/daily_round.py [hunk 3]
```diff
@@ -26,7 +26,7 @@ def filter_rounds_type(self, queryset, name, value):
...
         for v in values:
             try:
-                rounds_type.add(DailyRound.RoundsTypeDict[v])
+                rounds_type.add(DailyRound.RoundsType[v.upper()].value)
```
Comment (rithviknishad) [inline care/facility/api/viewsets/daily_round.py]:
```suggestion
                rounds_type.add(DailyRound.RoundsType[v].value)
```

File: care/facility/api/viewsets/patient.py (hunk 1 above)
Comment (rithviknishad) [reply care/facility/api/viewsets/patient.py:222]: This seems to be fine to me.

Comment (rithviknishad) [reply care/facility/api/viewsets/patient.py:222]: @sainak

**Review by rithviknishad — APPROVED** [review]

**Review by sainak — APPROVED** [review]

**Review by vigneshhari — APPROVED** [review]

**Pull request merged**